_posts/2016/2016-02-17-development-from-the-treches.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Development from the 'southern' trenches"
+date: "2016-02-17 09:00:00"
+icon: bookmark
+categories: working-at-epages
+authors: ["Roger"]
+---
+
+If you check our [epages](http://www.epages.com) website you will see that we have an office in Barcelona, WoW! Unlike other commercial offices we have a development team, double WoW!
+In this post I am going to explain what it means to live and work in Barcelona from an epages developer perspective.
+
+## Life in Barcelona
+Apart from the weather, culture, night life and such a big football team, [Barça](http://fcbarcelona.com) of course, Barcelona is much more than that. It is the capital of Catalonia, it has almost all the Gaudí heritage, astonishing beaches (like Costa Brava) and big mountains (Pyrenees) in less than 2 hours by car or train. Life is not just working ;)
+{% image trivia.jpg %}{% endimage %}
+
+### For developers
+As everybody knows Barcelona is one of the most important European cities in software development and hosts the Mobile World Congress, which brings amazing opportunities and projects to the city. This attracts talent which enriches the existing pool of [meet ups](http://www.meetup.com/cities/es/barcelona/) or technical events, where you can participate, learn or collaborate with your thoughts or ideas.
+
+## The office
+{% image bcn-barbecue.jpg 70% right %}{% endimage %}
+The office is located in the heart of Barcelona, in the Diagonal avenue, where it is full of shops, restaurants and good public transport connections. This excellent location offers us many different ways to come to the office in a decent time and choose what do you want to eat every day (Galician, French, Japanese or a mix of all of them!) or take away something to the office.
```

Comment:
and choose what do you want to eat every day > and choose what you'd like to eat every day